package.json

```diff
@@ -100,6 +100,6 @@
     "ember-truth-helpers": "^1.3.0"
   },
   "pr-bumper": {
-    "coverage": 94.22
+    "coverage": 93.17
```

Comment:
I would expect more tests added.

Reply:
misses for coverage are around the following type of logic:

if (this.isDestroyed || this.isDestroying) 

Unsure of a proper way to test this.

And having the dom fire scrolling events while the mouse is over the dropdown.